desktop_version/src/editor.cpp

```diff
@@ -2152,7 +2163,7 @@ bool editorclass::save(std::string& _path)
     {
         Script& script_ = script.customscripts[i];
 
-        scriptString += script_.name + ":|";
+        scriptString += script_.name + ":" + linebreak[0];
```

Comment:
See my other comment - if the user creates a new level and saves it, `linebreak` will be uninitialized here! (The best-case scenario is that they load a level beforehand, in which case it will be properly initialized, and then creating a new level will just carry over the `linebreak` from the previous level. Which is still bad, but at least it's not undefined behavior.)